lib/epicenter_web/templates/layout/_search_results.html.heex

```diff
@@ -0,0 +1,71 @@
+<div data-role="search-results" id="search-results">
+  <div id="search-results-inner">
+    <div>
+      <div id="search-close-results">
+        <a
+          data-role="close-search-results"
+          href="javascript:void(0);"
```

Comment:
Will probably not need this workaround in the future.

Reply:
added https://github.com/RatioPBC/epi-viewpoint/issues/15